1.Practicals_C/1.Decision_Making/Looping/Implementation/armstrong number

```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>  
+ int main()    
+{    
+    int n,r,sum=0,a;    
+    printf("enter a number =",n);    
+    scanf("%d",&n);    
+    a=n;    
+    while(n>0)    
+    {    
+        r=n%10;    
+        sum=sum+(r*r*r);    
```

Comment:
what if the armstrong number entered is four digit long?

Reply:
ok sir i will change this code for n digit number.